apps/cow-tools/src/lib/encode.ts

```diff
@@ -6,6 +7,32 @@ import {
 } from "./priceCheckersMappings";
 import { argType, PRICE_CHECKERS, PriceCheckerArgument } from "./types";
 
+export function encodePriceCheckerDataWithValidFromDecorator(
+  priceCheckerAddress: Address,
+  priceCheckerData: `0x${string}`,
+  validFrom: string,
+) {
+  const validFromDate = new Date(validFrom);
+  const validFromTimestamp = BigInt(validFromDate.getTime() / 1000);
```

Comment:
we have a function that does that on `packages/utils/date.ts` called `dateToEpoch`